src/ape/api/compiler.py

```diff
@@ -19,9 +16,6 @@ class CompilerAPI:
     this API.
     """
 
-    config: ConfigItem
-    """The :class:`ape.api.config.ConfigItem` for this compiler plugin."""
```

Comment:
Might want to keep this for now and initialized each instance with this property (still thinking about this)